bot/seasons/valentines/be_my_valentine.py

```diff
@@ -0,0 +1,237 @@
+
+
+        syntax: .bemyvalentine [user](optional) [p/poem/c/compliment/or you can type your own valentine message]
+        (optional)
+
+        example: .bemyvalentine (sends valentine as a poem or a compliment to a random user)
+
+        example: .bemyvalentine @Iceman#6508 p (sends a poem to Iceman)
+
+        example: .bemyvalentine @Iceman#6508 Hey I love you, wanna hang around ? (sends the custom message to Iceman)
+        """
+        if ctx.guild is None:
+            # This command should only be used in the server
+            msg = "You are supposed to use this command in the server."
+            return await ctx.send(msg)
+
+        channel = self.bot.get_channel(Lovefest.channel_id)
+
+        if user == ctx.author:
+            # Well a user cant valentine himself/herself.
+            await ctx.send('Come on dude, you cant send a valentine to yourself :expressionless:')
+
+        elif user is None:
+            # just making sure that the random does not pick up the same user(ctx.author)
+            USER_LOVEFEST.remove(ctx.author)
+            user = random.choice(USER_LOVEFEST)
+            USER_LOVEFEST.append(ctx.author)
+
+            if valentine_type is None:
+                # grabs a random valentine -can be a poem or a good message
+                valentine, title = self.random_valentine()
+                embed = discord.Embed(
+                    title=f"{EMOJI_1} {title} {user.display_name}{EMOJI_2}",
+                    color=Colours.pink
+                )
+
+            elif valentine_type in ['p', 'poem']:
+                valentine = self.valentine_poem()
+                embed = discord.Embed(
+                    title=f"{EMOJI_1} A poem dedicated to {user.display_name}{EMOJI_2}",
+                    color=Colours.pink
+                )
+
+            elif valentine_type in ['c', 'compliment']:
+                valentine = self.valentine_compliment()
+                embed = discord.Embed(
+                    title=f"{EMOJI_1} A compliment for {user.display_name}{EMOJI_2}",
+                    color=Colours.pink
+                )
+            else:
+                # in this case, the user decides to type his own valentine.
+                valentine = valentine_type
+                embed = discord.Embed(
+                    title=f'{EMOJI_1}A message for {user.display_name}{EMOJI_2}',
+                    color=Colours.pink
+                )
+
+            embed.description = f'{valentine} \n **{EMOJI_2}From {ctx.author}{EMOJI_1}**'
+            await channel.send(user.mention, embed=embed)
+
+    @commands.cooldown(1, 1800, BucketType.user)
+    @send_valentine.command(name='dm')
+    async def anonymous(self, ctx, user: typing.Optional[discord.Member] = None, *, valentine_type=None):
+        """
+    - This command DMs a valentine to be given anonymous to a user if specified or a random user having lovefest role.
```

Comment:
The leading hyphen should be removed. The docstring contents need to align with the quotes indenting.